Framework/Core/include/Framework/DataTypes.h

```diff
@@ -35,7 +35,8 @@ enum TrackTypeEnum : uint8_t {
 };
 enum TrackFlags {
   TrackTimeResIsRange = 0x1, // Gaussian or range
-  PVContributor = 0x2        // This track has contributed to the collision vertex fit
+  PVContributor = 0x2,       // This track has contributed to the collision vertex fit
+  OrphanTrack = 0x4          // Track has no association with any collision vertexd
```

Comment:
As you are on it, could you remove `ITSStandaloneTrack`  in line 32. I think it is not needed anymore.

Reply:
ok, done